<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1173</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Identifiant</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="id">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="m">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>90</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Masculin</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ville">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>230</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Tunis</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Beja</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gafsa</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="ins">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>290</y>
      <width>76</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Inscription</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ajouter">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>340</y>
      <width>82</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="c">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>420</y>
      <width>551</width>
      <height>192</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Identifiant</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>NÂ° Telephone</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Genre</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Ville</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Etat</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="aff_cli">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>340</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Afficher Clients</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Numero de telephone</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ntel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Genre</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="f">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>170</y>
      <width>90</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Feminin</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ville</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Etat</string>
    </property>
   </widget>
   <widget class="QListWidget" name="chan">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>90</y>
      <width>141</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="aff_chan">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>50</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Afficher Chance</string>
    </property>
   </widget>
   <widget class="QPushButton" name="aff_gag">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>40</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Afficher Gagnants</string>
    </property>
   </widget>
   <widget class="QListWidget" name="gag">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>80</y>
      <width>221</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1173</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
